CMakeLists.txt

```diff
@@ -132,6 +134,27 @@ else()
   message(WARNING "ZLIB not found - install zlib for xnpz file support")
 endif()
 
+message(STATUS "Trying to find HighFive for npz file support")
+find_package(HighFive)
+find_package(HDF5)
+if (${HighFive_FOUND} AND ${HDF5_FOUND})
+  message(STATUS "HighFive and HDF5 found, HDF5 file support enabled")
+  find_package(HDF5 REQUIRED)
+  get_target_property(HighFive_INCLUDE_DIRECTORIES_EXTRACTED HighFive INTERFACE_INCLUDE_DIRECTORIES)
+  target_include_directories(xtensor-io
+    INTERFACE
+      ${HighFive_INCLUDE_DIRECTORIES_EXTRACTED}
+  )
+  target_link_libraries(xtensor-io
+    INTERFACE
+      ${HDF5_LIBRARIES}
+  )
+else()
+  message(WARNING "HighFive and/or HDF5 not found - install HighFive and HDF5 for HDF5 file support")
+endif()
+
+target_link_libraries(xtensor-io ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
reminder: we should remove this (we thought it might fix the cmake with the new style HighFive linking).

Reply:
I'm not completely sure what to remove here ;) Line 156?

Reply:
yes, that line. we added it out of desperation :)